### PR TITLE
Bulk sample: Adds scenario for removing a property

### PR DESCRIPTION
**Pull request by j82w**
# Pull Request Template

## Description

This adds a sample on how to use bulk to remove a property from all the objects in a container.

## Type of change

Please delete options that are not relevant.

- [] Bug fix (non-breaking change which fixes an issue)
- [] New feature (non-breaking change which adds functionality)
- [] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [] This change requires a documentation update

## Closing issues

To automatically close an issue: closes #IssueNumber

### CODE REVIEW

File: Microsoft.Azure.Cosmos.Samples/Usage/BulkSupport/Program.cs [hunk 1]
```diff
@@ -149,7 +152,67 @@ private static async Task CreateItemsConcurrentlyAsync(Container container)
...
+            Console.WriteLine($"All items updated to remove property 'other'");
+        }
+
+        private static async Task RemoveItemProperty(
+            Container container,
+            JObject item)
+        {
+            ItemRequestOptions itemRequestOptions = new ItemRequestOptions()
+            {
+                EnableContentResponseOnWrite = false,
```
Comment (kirankumarkolli) [inline Microsoft.Azure.Cosmos.Samples/Usage/BulkSupport/Program.cs:192]: As a pattern isn't it better to have it at client scope?

File: Microsoft.Azure.Cosmos.Samples/Usage/BulkSupport/Program.cs [hunk 2]
```diff
@@ -149,7 +152,67 @@ private static async Task CreateItemsConcurrentlyAsync(Container container)
...
+
+            while (true)
+            {
+                item.Remove("other");
+                string id = item["id"].Value<string>();
+                string pk = item["pk"].Value<string>();
+                itemRequestOptions.IfMatchEtag = item["_etag"].Value<string>();
+                try
+                {
+                    await container.ReplaceItemAsync<JObject>(item, id, new PartitionKey(pk));
```
Comment (kirankumarkolli) [inline Microsoft.Azure.Cosmos.Samples/Usage/BulkSupport/Program.cs:203]: requestoption missing right?

File: Microsoft.Azure.Cosmos.Samples/Usage/BulkSupport/Program.cs [hunk 3]
```diff
@@ -149,7 +152,67 @@ private static async Task CreateItemsConcurrentlyAsync(Container container)
...
+            JObject item)
+        {
+            ItemRequestOptions itemRequestOptions = new ItemRequestOptions()
+            {
+                EnableContentResponseOnWrite = false,
+            };
+
+            while (true)
+            {
+                item.Remove("other");
```
Comment (kirankumarkolli) [inline Microsoft.Azure.Cosmos.Samples/Usage/BulkSupport/Program.cs:197]: nit: Make it isolated (with new line and a comment)

File: Microsoft.Azure.Cosmos.Samples/Usage/BulkSupport/Program.cs [hunk 4]
```diff
@@ -149,7 +152,67 @@ private static async Task CreateItemsConcurrentlyAsync(Container container)
...
+                try
+                {
+                    await container.ReplaceItemAsync<JObject>(item, id, new PartitionKey(pk));
+                    return;
+                }
+                catch (CosmosException ex)
+                {
+                    Console.WriteLine($"Replace item failed at {DateTime.UtcNow} with id:{id}; pk:{pk}; Excepion: {ex}");
+                    // The item was updated after the query. Read the latest item and try update again.
+                    if (ex.StatusCode == HttpStatusCode.Conflict)
```
Comment (kirankumarkolli) [inline Microsoft.Azure.Cosmos.Samples/Usage/BulkSupport/Program.cs:210]: Silently eating non-conflict ones. Prefer conditional catch

**Review by kirankumarkolli — DISMISSED** [review]

File: Microsoft.Azure.Cosmos.Samples/Usage/BulkSupport/Program.cs (hunk 1 above)
Comment (j82w) [reply Microsoft.Azure.Cosmos.Samples/Usage/BulkSupport/Program.cs:192]: Each item is setting the etag value so it can't be at the client scope.

**Review by ealsur — APPROVED** [review]
LGTM, thanks!

**Pull request merged**